test/unit/metadata_test.py

```diff
@@ -232,6 +233,28 @@ def test_frictionless_data_package_resources_populated():
         )
 
 
+def test_merge_descriptions():
+    "Ensure descriptions are merged properly."
+    left = {
+        "additional_summary_text": "one",
+        "additional_details_text": "two",
+        "usage_warnings": ["red"],
+    }
+    right = {
+        "additional_summary_text": "fish",
+        "additional_details_text": "poisson",
+        "usage_warnings": ["blue"],
+    }
+    result = {
+        "additional_summary_text": "one fish",
+        "additional_details_text": "two\n\npoisson",
+        "usage_warnings": ["red", "blue"],
+    }
+    assert merge_descriptions(left, right) == result
+    # make sure we didn't accidentally modify the source list during the merge
+    assert left["usage_warnings"] == ["red"]
```

Comment:
this line is a regression test; my first draft just kept appending harvest warnings onto the shared 923 dicts until there were like 8 copies